src/release/rollups.md

```diff
@@ -23,67 +23,99 @@ queue has been merged.
+Very old (several months) or very large PRs that are extremely prone to merge conflicts may also be given a slight priority bump (`p=1`) to finally get them out of the queue without having to rebase them repeatedly.
+Ultimately, we want to keep the number of regressions per merge at a minimum while also minimizing the amount of time between approval and the final merge of a PR, to avoid unnecessary merge conflicts and rebases.
 
 
 ## Failed rollups
 If the rollup has failed, run the `@bors retry` command if the
-failure was spurious (e.g. due to a network problem or a timeout). If it wasn't spurious,
-find the offending PR and throw it out by copying a link to the rust-logs-analyzer comment,
-and writing `Failed in <link_to_comment>, @bors r-`. Hopefully,
-the author or reviewer will give feedback to get the PR fixed or confirm that it's not
-at fault. The failed rollup PR can be closed.
+failure was spurious (e.g. due to a network problem or a timeout).
+There may be a matching `CI-spurious-fail-.*` label that you can use to tag the PR as such, to help discover common fail patterns.
+If it wasn't spurious, find the offending PR and throw it out by copying a link to the rust-logs-analyzer comment,
+and leaving a comment like `Failed in <link_to_comment>, @bors r-`.
+In case the log-analyzer does not give any meaningful output, you can directly open the ci-logs (the `(web)` link), find the point where the error was thrown
+and directly copy the URL to the respective line in the log output.
+Hopefully, the author or reviewer will give feedback to get the PR fixed or confirm that it's not
+at fault. The failed rollup PR should then be closed.
+
+Once you've removed the offending PR, recreate your rollup without it (see 1.).
+Merge one batch of PRs by throwing out the failures one by one instead of adding new PRs to it, as this may introduce additional points of failure.
 
-Once you've removed the offending PR, re-create your rollup without it (see 1.).
 Sometimes however, it is hard to find the offending PR. If so, use your intuition
 to avoid the PRs that you suspect are the problem and recreate the rollup.
 Another strategy is to raise the priority of the PRs you suspect,
 mark them as `rollup=never` (or `iffy`) and let bors test them standalone to dismiss
-or confirm your hypothesis.
+or confirm your hypothesis, or split the rollup into 2 smaller ones until are certain of the failure cause. If a PR was found to be the cause and other PRs were "wrongfully" `iffy`'d, they can of course be reprioritised as `maybe` again.
 
-If a rollup continues to fail you can run the `@bors rollup=never` command to
+If a PR in a rollup continues to fail you can run the `@bors rollup=never` command to
 never rollup the PR in question.
```

Comment:
```suggestion
If a PR in a rollup continues to fail you can run the `@bors rollup=never` command to
ensure the PR gets tested independently, since it's likely it will fail again in the future.
```